### DISCUSSION TITLE
The Copilot Suggestions Different In Different Editors

### DISCUSSION THREAD

**Discussion by wewrere41** [community/community#10612] [Copilot Conversations]
Rider 2021.3.2, C#10
Copilot never suggest more than one solution in Rider. But for example same code in vsCode, copilot works correctly.
![rider64_GEleoSIlIe](https://user-images.githubusercontent.com/45511976/151315056-4ff38989-ffab-49b5-9240-925fe668272c.png)
![image](https://user-images.githubusercontent.com/45511976/151657784-264a6f93-d27a-4e2d-94bc-3f8f29d6d725.png)
![image](https://user-images.githubusercontent.com/45511976/151657786-7918b55c-5a2c-485f-a733-c6a096754535.png)

**Comment by AcmeGamers** [comment]
I believe this is normal since it is just a preview version and more code will be added on later. If you want to confirm, you can make a simple JS file and write `function calculateDatesBetweenDays()`, it will show you 10 answers you can choose from. Hope this helps

**Reply by wewrere41** [reply]
Thank you. I actually wrote this by comparing it to this video. When he writes, more than one suggestion appears, when I write the same thing, only 1 suggestion appears. I couldn't understand why. Could it be evolving according to the user?
https://youtu.be/XceIKXeIrD0?t=127
[](https://youtu.be/XceIKXeIrD0?t=127)

**Reply by AcmeGamers** [reply]
I believe that it may haven't fetched data fully, and it also could be evolving by the user (I'm not fully sure about the evolution, but it can be the case since it is AI). Although I made a test from the images you gave above, I got 5 outputs. Give it a try now, I hope it works for you now 🙂

![image](https://user-images.githubusercontent.com/52010365/151654324-86fb21a6-35d8-4b4a-8a6d-d01712c432e4.png)

![image](https://user-images.githubusercontent.com/52010365/151654293-b5e65dda-69ae-4532-a709-0b0800675d08.png)

**Reply by wewrere41** [reply]
![image](https://user-images.githubusercontent.com/45511976/151654968-09bae549-f6b3-4071-b849-130f7f118642.png)

I get one suggestion again :(

**Reply by AcmeGamers** [reply]
Try pressing this, or you can reopen the editor once again. If that still doesn't work, try reinstalling the extension

![image](https://user-images.githubusercontent.com/52010365/151654997-aa8a5dba-a042-4198-906b-9efcc0eca075.png)

**Reply by wewrere41** [reply]
I tried this solutions again and again :(

**Reply by AcmeGamers** [reply]
Even reinstalling the extension didn't work owo;?

**Reply by wewrere41** [reply]
Yes , didnt work. :(

**Reply by AcmeGamers** [reply]
Hmm, this is weird, can you check this in VS Code?

**Reply by wewrere41** [reply]
![image](https://user-images.githubusercontent.com/45511976/151657784-264a6f93-d27a-4e2d-94bc-3f8f29d6d725.png)
![image](https://user-images.githubusercontent.com/45511976/151657786-7918b55c-5a2c-485f-a733-c6a096754535.png)

Its working normal in vsCode
Rider, getting suggestion slower (sometimes giving timeout) and 99 percent of cases giving just one solution.

**Comment by wewrere41** [comment]
Any suggestion?
